pyvxl/uds.py

```diff
@@ -207,13 +218,257 @@ def ecu_reset(self, reset_type, raise_error=True, **kwargs):
             result = data
         return result
 
-    def clear_dtcs(self, *args, **kwargs):
+    def clear_dtc_info(self, dtc_group, raise_error=True, **kwargs):

```

Comment:
Could you update the docstrings to follow google style? It looks like most of them are but a few were missed like this function